### PR TITLE
tamsyn: init at 1.11

### PR DESCRIPTION
**Pull request by robertseaton**
###### Motivation for this change

Wanting to look like a l33t hackerman at Starbucks, needing a suitable bitmap font. 

###### Things done

- Created build file `fonts/tamsyn/default.nix`
- Edited `all-packages.nix`to add package
- Edited `maintainer-list.nix` to add self as maintainer

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dtzWill** [comment]
LGTM! :)

File: pkgs/data/fonts/tamsyn/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl, mkfontdir, mkfontscale }:
+
+let
+  version = "1.11"; 
+in stdenv.mkDerivation rec {
```
Comment (ryantm) [inline pkgs/data/fonts/tamsyn/default.nix]:
```suggestion
in stdenv.mkDerivation {
```
If you are going to have version in the let block, you probably don't need a recursive attrset.

File: pkgs/data/fonts/tamsyn/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  name = "tamsyn-font-${version}";
```
Comment (ryantm) [inline pkgs/data/fonts/tamsyn/default.nix]:
```suggestion
  pname = "tamsyn-font";
  inherit version;
```

File: pkgs/data/fonts/tamsyn/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    mkfontscale
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "13l7ighfmn3kmqmchlksfg8ss22ndjk71rs0f9fn5p5zk7s4dn5x";
+
+  meta = with stdenv.lib; {
+    description = "A monospace bitmap font aimed at programmers";
+    longDescription = '' Tamsyn is a monospace bitmap font, primarily aimed at
```
Comment (ryantm) [inline pkgs/data/fonts/tamsyn/default.nix]:
```suggestion
    longDescription = ''Tamsyn is a monospace bitmap font, primarily aimed at
```

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by ryantm** [comment]
@GrahamcOfBorg build tamsyn

**Review by ryantm — APPROVED** [review]

**Comment by ryantm** [comment]
@GrahamcOfBorg build tamsyn

**Comment by ryantm** [comment]
thanks!

**Pull request merged**